Content.Server/Light/Components/PoweredLightComponent.cs

```diff
@@ -28,6 +28,9 @@ public class PoweredLightComponent : Component
         [DataField("hasLampOnSpawn")]
         public bool HasLampOnSpawn = true;
 
+        [DataField("particularLampOnSpawn")]
```

Comment:
```suggestion
        [DataField("particularLampOnSpawn", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
```
to have the id validated

Reply:
Seems better to just make the particularLampOnSpawn nullable and combine with HasLampOnSpawn?

Reply:
yes, that too

Reply:
So, it would be preferable for me to get rid of the HasLampOnSpawn boolean entirely, and have any fixture that wants a lamp specify it explicitly, in addition to the suggested change (which if I understand it correctly, makes strings that aren't valid prototypes fail the YAML linter)? I can do that.

Reply:
I've made changes as requested, however there's an issue that may uncover a problem with the YAML linting itself. The new version works as intended, but YAML verification still fails. Apparently, explicitly resetting this nullable data field to null in YAML after it's set to something by a parent prototype fails the suggested constraint, as can be seen in the updated version.

Reply:
If there's no other suggestions, I can try restructuring the YAML so that filled lights inherit from empty ones, and not the other way around. That should get equivalent behaviour without triggering this issue.

Reply:
Restructured YAML to avoid having to explicitly declare nulls. Should this be a feature our YAML parsing supports properly at some point in the future?